cloud/blockstore/libs/nbd/device.h

```diff
@@ -20,6 +20,17 @@ struct IDeviceConnection
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct IDeviceConnectionFactory
```

Comment:
We can do without this interface:
```cpp
using TDeviceConnectionFactory = std::function<
    IDeviceConnectionPtr(TNetworkAddress connectAddress, TString deviceName)>;
```

Reply:
discussed in private, decided not to change